lib/Runtime/Language/InlineCache.h

```diff
@@ -964,6 +964,70 @@ namespace Js
         void Unregister(ScriptContext * scriptContext);
     };
 
+    class IsConcatSpreadableCache
+    {
+        Type *type0, *type1;
+        int lastAccess;
+        BOOL result0, result1;
+
+    public:
+        IsConcatSpreadableCache()
+        {
+            type0 = nullptr;
```

Comment:
Will change in next rev.

---
In reply to: [68425816](https://github.com/Microsoft/ChakraCore/pull/1195#discussion_r68425816) [](ancestors = 68425816)